src/wallet/wallet.cpp

```diff
@@ -986,7 +986,7 @@ void CWallet::MarkConflicted(const uint256& hashBlock, int conflicting_height, c
     auto locked_chain = chain().lock();
     LOCK(cs_wallet);
 
-    int conflictconfirms = -locked_chain->getBlockDepth(hashBlock);
+    int conflictconfirms = (m_last_block_processed_height - conflicting_height + 1) * -1;
```

Comment:
36b68de5b2938722911db900ca299f7008780d01

What's the problem with `-(m_last_block_processed_height - conflicting_height + 1)`?